### PR TITLE
[Worker] Initial Setup

### PR DESCRIPTION
**Pull request by MNThomson**
This is the initial setup for the worker node's program

### CODE REVIEW

**Review by Addie-Finke — DISMISSED** [review]
Looks all good to me

File: src/worker/vm.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	var firecrackerBinary string
+	firecrackerBinary, err = exec.LookPath("firecracker")
+	if os.IsNotExist(err) {
+		return fmt.Errorf("binary %q does not exist: %v", firecrackerBinary, err)
+	}
+	if err != nil {
+		return fmt.Errorf("failed to start binary, %q: %v", firecrackerBinary, err)
+	}
+
+	m, err := firecracker.NewMachine(vmmCtx, fcCfg, machineOpts...)
```
Comment (Redskulling) [inline src/worker/vm.go]: Bad variable name ```m``` should be something like ```machine```.

File: src/worker/vm.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	log.Printf("Machine started successfully")
+	return nil
+}
+
+// Create custom signal handlers
+func signalHandlers(ctx context.Context, m *firecracker.Machine) {
+	go func() {
+		// Reset signal handlers
+		signal.Reset(os.Interrupt, syscall.SIGTERM, syscall.SIGQUIT)
+		c := make(chan os.Signal, 1)
```
Comment (Redskulling) [inline src/worker/vm.go]: Same with ```c```, which I assume to be a channel.

**Review by Redskulling — COMMENTED** [review]
Other than whats addressed in my comments, looks good.

**Review by Redskulling — APPROVED** [review]
Looks good.

**Pull request merged**